packages/shared/src/components/feeds/FeedNav.tsx

```diff
@@ -152,27 +137,6 @@ function FeedNav(): ReactElement {
             <Tab key={label} label={label} url={url} />
           ))}
         </TabContainer>
-
```

Comment:
which one is this we're removing, I just can't seem to recall?

Reply:
I should revert this. Give me one moment. Reference: https://dailydotdev.slack.com/archives/C02AA18RD1D/p1727856011408519